components/wpt-runs.html

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+
+<dom-module id="wpt-runs">
+  <template>
+    <style>
+      table {
+        width: 100%;
+      }
+      tr td {
+        padding: 0 0.5em;
+        margin: 2px;
+      }
```

Comment:
Can you add:

```css
a {
  text-decoration: none;
  color: #0d5de6;
  font-family: monospace;
}
```

(it's from the `<wpt-results>` component - I don't know a better way of sharing CSS between components :crying_cat_face:)

Reply:
Done.